### ISSUE TITLE
[Bug]: Uncaught ReferenceError: currentlyReorderingStatus is not defined

### ISSUE THREAD

**Issue by bdelamatre** [rappasoft/laravel-livewire-tables#2069]
### What happened?

Console throws uncaught error: "Uncaught ReferenceError: currentlyReorderingStatus is not defined"

### How to reproduce the bug

Check console after page load where livewire table component is used

### Package Version

3.5.2

### PHP Version

8.3.x

### Laravel Version

11.x

### Alpine Version

latest

### Theme

Tailwind 3.x

### Notes

I believe the issue is simply because currentlyReorderingStatus isn't defined on the table's x-data component, but I haven't tested as that would require me to setup an environment to perform a build. A quick look at the JS leads me to thing that the following line of code should be added to `tableWrapper.js`

```
currentlyReorderingStatus: wire.entangle('currentlyReorderingStatus'),
```

### Error Message

Uncaught ReferenceError: currentlyReorderingStatus is not defined

**Comment by lrljoe** [comment]
If you have previously published the views then try without publishing them

**Comment by johnanortega** [comment]
Downgrade LiveWire to 3.0.0

**Comment by lrljoe** [comment]
Upgrade to latest release, which has a fix in it for lazily loaded tables

**Comment by bdelamatre** [comment]
Confirmed this is resolved in 3.5.5. Thank you @lrljoe !
